README.md

```diff
@@ -207,7 +207,21 @@ get_current_span().add_event('<error-message>', {'lumigo.type': '<error-type>'})
 | --- | --- | :---: | :---: | :---: | :---: | :---: |
 | | | 3.7 | 3.8 | 3.9 | 3.10 | 3.11 |
 | botocore | [boto3](https://pypi.org/project/boto3) | 1.17.22~1.28.40|1.17.22~1.28.40|1.17.22~1.28.40|1.17.22~1.28.40|1.17.22~1.28.40|
-| django | [django](https://pypi.org/project/django) |  |4.2.5|4.2.5|4.2.5|4.2.5|
+| django | [django](https://pypi.org/project/django) | 3.2.1~3.2.21|3.2.1~3.2.21|3.2.1~3.2.21|3.2.1~3.2.21|3.2.1~3.2.21|
```

Comment:
Should you be manually changing the table? shouldn't the version testing job update the table?

Reply:
this is automatically updated when the checks script is run. this is to ensure that if you have locally modified the tested versions files (or added new ones) the README is updated accordingly. in this case, i manually updated the tested versions files to verify that the django tests are working